charts/nextcloud/README.md

```diff
@@ -96,7 +96,7 @@ The following table lists the configurable parameters of the nextcloud chart and
 | `ingress.path`                                             | The `Path` to use in Ingress' `paths`                                                  | `/`                        |
 | `ingress.pathType`                                         | The `PathType` to use in Ingress' `paths`                                              | `Prefix`                   |
 | `ingress.tls`                                              | Ingress TLS configuration                                                              | `[]`                       |
-| `nextcloud.host`                                           | nextcloud host to create application URLs                                              | `nextcloud.kube.home`      |
+| `nextcloud.host`                                           | nextcloud host to create application URLs, updates trusted_domains at installation time only| `nextcloud.kube.home` |
```

Comment:
Yes it seems this is a limitation of the container. At least it was the behavior it monitored.

Reply:
We have other variables that behave the same way, for example the database settings. Those settings are only copied at install time which is obviously pretty bad in a kubernetes environment. I wanted to open a PR for that but haven't found the time yet.

Reply:
@provokateurin  Yes, the way the nextcloud docker container works is more than ideal. Its logic is more made from "maintain nextclould on a server" prosective in mind, instead of an idempotent container. e.g. installing nextcloud instead of having it already in place. 
Futher i would like to see a more modular entrypoint, that allows to write all config in a separate step. This would allow to use an init container with root privileges to e.g. write the php-redis config and then run the main container without root.

Reply:
There is `nextcloud.configs` which can be used to write custom configs. Those would be evaluated at chart install time. This is the way to properly implement it. I think this discussion should happen in a new issue as it is not directly related to this PR and also concerns more of the configs than just the trusted domains.